code/controllers/configuration/configuration.dm

```diff
@@ -350,7 +350,9 @@ Example config:
 			if ("map")
 				currentmap = load_map_config(data, MAP_DIRECTORY_MAPS)
 				if(currentmap.defaulted)
-					log_config("Failed to load map config for [data]!")
+					var/error_message = "Failed to load map config for [data]!"
+					log_config(error_message)
+					log_mapping(error_message, TRUE)
```

Comment:
it may be better to explicitly do `skip_world_log = TRUE` since it's hard to tell what the boolean does in this statement without having to go to the defined code.